src/components/Header.js

```diff
@@ -15,31 +15,34 @@ const Header = ({ location }) => (
     <div className='md-flex flex-auto items-baseline px2 sm-px6'>
       <div className='flex-auto'>
         <div className='inline-block'>
-          <span className='mb1 h6 caps bold line-height-1 blue-gray'>
+          <span
+            className='mb1 h6 caps bold line-height-1 blue-gray block'
+          >
             Federal Bureau of Investigation
           </span>
-          <br />
-          <span className='h2 serif line-height-1'>Crime Data Explorer</span>
+          <a className='h2 serif line-height-1 white' href='/'>
+            Crime Data Explorer
+          </a>
         </div>
       </div>
       <div className='mxn1 mt1 md-m0 truncate'>
         <Link
           to='/'
-          className='mx1 h4 white'
+          className='mx2 h4 white'
```

Comment:
if this increases to `mx2`, then the parent div must be changed to `mxn2` so that the right link is flush with the end of the container